### PR TITLE
Add Jenkins file for builds on ci.jenkins.io

### PR DESCRIPTION
**Pull request by oleg-nenashev**
@reviewbybees

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
+// TODO: Explicitly require LTS baseline once SSHD Module 2.x gets into LTS
```
Comment (jglick) [inline Jenkinsfile]: What does this mean? By default we build against whatever baseline is listed in this module’s POM. You only need to patch `Jenkinsfile` if you wish to build & test against multiple/alternate baselines.

Comment (oleg-nenashev) [inline Jenkinsfile]: I do. My plan it to test...

* Minimal supported LTS version by the module
* Latest supported LTS version 
* Latest weekly
* Maybe other particular LTS lines I am interested in

**Comment by oleg-nenashev** [comment]
@reviewbybees done

**Comment by oleg-nenashev** [comment]
Thanks @mc1arke and @jglick !

**Pull request merged**